test/configCases/library/0-create-library/self-reference-cjs.js

```diff
@@ -0,0 +1,3 @@
+console.log(exports)
+console.log(module)
+console.log(this)
```

Comment:
Can we avoid using `console.log`, it outputs a lot of span into CI output, ideally we should remove `console.log` and `debugger` from all tests (except where we need it)